odoo/tools/mimetypes.py

```diff
@@ -170,7 +170,11 @@ def guess_mimetype(bin_data, default='application/octet-stream'):
 
     # magic from pypi https://pypi.python.org/pypi/python-magic/
     if hasattr(magic,'from_buffer'):
-        guess_mimetype = lambda bin_data, default=None: magic.from_buffer(bin_data, mime=True)
+        def guess_mimetype(bin_data, default=None):
+            mimetype = magic.from_buffer(bin_data, mime=True)
+            if mimetype and mimetype == 'image/svg':
+                mimetype += '+xml'
```

Comment:
I guess for the final word either security or framework python.

Just as I pass by I would suggest to return the full type `image/svg+xml` for clarity instead of building it dynamically.

Reply:
:+1: